code/__HELPERS/traits.dm

```diff
@@ -96,10 +96,14 @@
 			} \
 \
 			for(var/TRAIT in target.status_traits) { \
+				if(!target.status_traits[TRAIT]) \
+					continue; \
```

Comment:
So basically, at the start of the TM we had a major issue where during this loop it would say, remove hands blocked, which would then remove pull blocked. 
But the loop would continue, get to pull blocked and try and remove it, runtiming, causing a whole bunch of issues.